Sources/muterCore/CLICommands/RunCommand/RunCommandHandler.swift

```diff
@@ -0,0 +1,37 @@
+@available(OSX 10.13, *)
+class RunCommandHandler {
```

Comment:
just want to confirm....
when we build with the optimization flag, will the compiler mark that as final for us?